front/src/components/mmt-ticket-route.component.vue

```diff
@@ -0,0 +1,70 @@
+﻿<template>
+  <div class="route">
+    <label class="routeDate text-body-regular">{{ routeItem.date }}</label>
+    <br/>
+    <div class="h-centered route-line">
+      <label class="departureTime text-small text-semi-bold">{{ routeItem.departureTime }}</label>
+      <span class="line"/>
+      <label class="arrivalTime text-small text-semi-bold">{{ routeItem.arrivalTime }}</label>
+    </div>
+    <br/>
+    <div class="h-centered route-line not-accented">
+      <label class="departureCode text-small">{{ routeItem.departureCode }}</label>
+      <span class="travelTime" >
+        <label class="text-small">В пути</label>
+        <br/>
+        <label class="text-small">{{ routeItem.hours }} час {{ routeItem.minutes }} минут</label>
+      </span>
+      <label class="arrivalCode text-small">{{ routeItem.arrivalCode }}</label>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'mmt-ticket-route',
+  props: {
+    routeItem: Object
+  }
+};
+</script>
+
+<style scoped>
+  .route {
+    margin-top: 4px;
+    margin-left: 12px;
+    margin-right: 12px;
+    height: 72px;
+    position: relative;
+  }
+  .routeDate {
+    float: right;
+  }
+  .arrivalTime, .arrivalCode {
+    float: right;
+  }
+  .h-centered {
+    position: absolute;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%)
+  }
+  .route-line {
+    display: flex;
+    align-items: center;
+    margin-top: 12px;
+  }
+  .line {
+    height: 1px;
+    margin: 8px;
+    padding: 0;
+    width: 100px;
+    border-top: 1px solid var(--mmt-color-light-gray);
+  }
+  .travelTime {
```

Comment:
.travel-time
please, use lower kebab-case
https://github.com/horba/my-trip/wiki/Frontend-Namings-and-Structure